api_tests/utils.py

```diff
@@ -25,3 +25,6 @@ def urlparse_drop_netloc(url):
     if url[4]:
         return url[2] + '?' + url[4]
     return url[2]
+
+def assert_dict_contains_subset(a, b):
```

Comment:
This should probably go in `tests.base`, since there's nothing api-specific about it.